DesignSafe-CI/portal
# conf: Prevent non-production deployments from being indexed by search engines

## Overview: ##
Block search engines from indexing pages if `DESIGNSAFE_ENVIRONMENT` is not set to `default` (i.e. production). See: https://developers.google.com/search/docs/crawling-indexing/block-indexing
## PR Status: ##


## Summary of Changes: ##

## Testing Steps: ##
1. Access Designsafe at a non-production deployment
2. Check that `<meta name="robots" content="noindex">` is in the HTML head.

## UI Photos:

## Notes: ##
